content/community/links.html

```diff
@@ -9,9 +9,7 @@
 <p><strong>Social Networking</strong></p>
 
 <ul>
- <li><a href="https://plus.google.com/communities/116050756631351429142" title="Google Plus Community Group">Google Plus Community Group</a></li>
  <li><a href="https://www.facebook.com/groups/haikuosgroup/" title="Facebook Community Group">Facebook Community Group</a></li>
- <li><a href="https://www.linkedin.com/groups?gid=165215" title="Linkedin Community Group">Linkedin Community Group</a></li>
```

Comment:
Correct link appears to be https://www.linkedin.com/groups/165215/

Reply:
Done, thanks!